src/fix/main.cpp

```diff
@@ -1451,6 +1445,55 @@ int main(int argc, char *argv[]) {
 	argv += musl_optind;
 	bool failed = nbErrors;
 
+	if (logoFilename && (fixSpec & (FIX_LOGO | TRASH_LOGO))) {
```

Comment:
@ISSOtm Should we read the custom logo file unconditionally, or only if we'll be using it? If you pass `-L nonexistent.1bpp` without passing `-f l` (or `-f L`, or `-v`), should rgbfix give an error?

Reply:
I think it's fair to go either way. So I'd break that tie by whatever is the simplest for us. (Thus, unconditionally reading it.)